config/locales/two_factor_authentication/es.yml

```diff
@@ -24,7 +24,7 @@ es:
       instructions: Si no tiene acceso a otro dispositivo, guarde bien sus códigos de
         respaldo. No podrá iniciar sesión en %{app_name} si pierde sus códigos
         de respaldo.
-      new_backup_codes_html: Necesito nuevos códigos de respaldo
+      new_backup_codes_html: Necesito nuevos códigos de respaldo&ZeroWidthSpace;
```

Comment:
If you think that's a better way to prevent wrapping, I'm all for it? I didn't have a sense of the font size/layout of this page

Reply:
What do you think about the idea of the new test only running over the `en.yml` file, or just checking that _at least one_ of the locales has HTML? So that we don't need to change this file.

Reply:
I like the idea of checking that "at least one" has HTML... but the current structure of the assertions is each file independently, not grouped by locale. Let me noodle on that

Reply:
Gave it a shot in a300659cd1fd0d2f1cabff080b9e3e943c46cef6, PTAL!

I futzed with it, so a failure where one string has HTML but none of them should looks like this now:
```
Failures:

  1) I18n config/locales/two_factor_authentication has HTML inside at least one locale string for all keys with .html or _html 
     Failure/Error: expect(bad_keys).to be_empty
       expected `{"two_factor_authentication.account_reset.cancel_link"=>{"en"=>"Cancel your request", "es"=>"Cancelar su solicitud <STRONG>", "fr"=>"Annuler votre demande"}}.empty?` to be truthy, got false
```